.github/workflows/ci.yml

```diff
@@ -40,15 +23,18 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: '3.8'
-      - run: sudo pip install black==24.3.0 ruff==0.1.9
+      - run: sudo pip install black==23.1.0 ruff==0.0.287
       - run: ruff check .
       - run: black --check .
 
   backend-unit-tests:
     runs-on: ubuntu-22.04
     needs: backend-lint
     env:
-      FRONTEND_BUILD_MODE: 0
+      COMPOSE_FILE: .ci/compose.ci.yaml
```

Comment:
Missed a sub-directory? :smile:

Reply:
Ah, no. I am staging this as two separate commits. I'll post the second so you can see what I'm trying to do.

Reply:
No worries. :smile: